main.R

```diff
@@ -395,6 +395,66 @@ prep_key_bars_portfolio <-
   }
 
 
+# -------------------------------------------------------------------------
+# prep_emissions_trajectory ------------------------------------------------------
+# based on pacta.portfolio.report:::prep_emissions_trajectory, but does not 
+# filter by scenario and scenario source
+
+prep_emissions_trajectory <-
+  function(equity_results_portfolio,
+           bonds_results_portfolio,
+           portfolio_name,
+           pacta_sectors,
+           year_span,
+           start_year
+  ) {
+    emissions_units <-
+      c(
+        Automotive = "tons of CO\U00002082 per km per cars produced",
+        Aviation = "tons of CO\U00002082 per passenger km per active planes",
+        Cement = "tons of CO\U00002082 per tons of cement",
+        Coal = "tons of CO\U00002082 per tons of coal",
+        `Oil&Gas` = "tons of CO\U00002082 per GJ",
+        Power = "tons of CO\U00002082 per MWh",
+        Steel = "tons of CO\U00002082 per tons of steel"
+      )
```

Comment:
NB/ unrelated to this PR but.... this seems very likely to fall out of sync 😅

Reply:
Agreed, but I'm not sure where to put it. It was baked-in to `pacta.portfolio.report:::prep_emissions_trajectory()`, so ported over here, unless/until it's available somewhere in a better format.